### DISCUSSION TITLE
Add support for vacuums

### DISCUSSION THREAD

**Discussion by patrik-malina** [andrei-tatar/node-red-contrib-smartnora#207] [Ideas]
Hi @andrei-tatar, can you add support for https://developers.google.com/assistant/smarthome/guides/vacuum?

Thanks.

**Comment by andrei-tatar** [comment]
@patrik-malina I think the API already supports it (if I remember correctly). I will need to check in the upcoming weeks. It's a bit busy over here right now.

What are the traits useful to you for the vacuum?

**Reply by patrik-malina** [reply]
Start/stop and dock would be essential.
Energy level and locate could be optional...

**Reply by andrei-tatar** [reply]
@patrik-malina latest version comes with support for vacuum. Let me know if there are any issues

**Comment by patrik-malina** [comment]
Thank you. Will report.

**Comment by patrik-malina** [comment]
It's working as it should. Thank you for your effort. 
Renewed subscription :)
